funlet-simple-menu/tests/funlet-simple-menu.test.js

```diff
@@ -240,14 +240,11 @@ test('[SIMPLE-MENU-OUTPUT-GATHER-DIGITS-0] Gather Digits Without Message', () =>
   const EMPTY_MESSAGE = '';
   const DIGITS = 5;
   const GATHER_DIGITS =
-    XML_DECLARATION +
-    '<Response>' +
-    '<Gather numDigits="' +
-    DIGITS +
-    '"/>' +
-    '</Response>';
-
-  let response = new Twilio.twiml.VoiceResponse();
+    `${XML_DECLARATION}<Response>` +
```

Comment:
all these multiline strings can be written to use a single starting \` and an ending `